pkg/gds/gds.go

```diff
@@ -375,7 +378,11 @@ func (s *GDS) Verification(ctx context.Context, in *api.VerificationRequest) (ou
 		}
 
 		if len(vasps) != 1 {
-			log.Warn().Str("common_name", in.CommonName).Int("nresults", len(vasps)).Msg("wrong number of VASPs returned from search")
+			if len(vasps) > 1 {
+				// Don't warn when common name is not found, just when multiple results are returned
+				log.Warn().Str("common_name", in.CommonName).Int("nresults", len(vasps)).Msg("multiple VASPs returned from common name search in verification")
```

Comment:
Thank you for adding "in verification" to the string here so that we can differentiate it from the similar `Lookup` case!